nncf/config/schemata/algo/quantization.py

```diff
@@ -0,0 +1,512 @@
+                                                         default=WEIGHTS_QUANT_START_EPOCH),
+            "lr_poly_drop_start_epoch": with_attributes(NUMBER,
+                                                        description="Epoch to start dropping the learning rate. If "
+                                                                    "unspecified, learning rate will not be dropped."),
+            "lr_poly_drop_duration_epochs": with_attributes(NUMBER,
+                                                            description="Duration, in epochs, of the learning "
+                                                                        "rate dropping process.",
+                                                            default=LR_POLY_DURATION_EPOCHS),
+            "disable_wd_start_epoch": with_attributes(NUMBER,
+                                                      description="Epoch to disable weight decay in the optimizer. "
+                                                                  "If unspecified, weight decay will not be disabled."),
+            "base_lr": with_attributes(NUMBER, description="Initial value of learning rate.",
+                                       default=STAGED_QUANTIZATION_BASE_LR),
+            "base_wd": with_attributes(NUMBER, description="Initial value of weight decay.",
+                                       default=STAGED_QUANTIZATION_BASE_WD),
+        },
+        "additionalProperties": False
+    }
+}
+
+QUANTIZATION_ALGO_NAME_IN_CONFIG = "quantization"
+QUANTIZATION_PRESETS_SCHEMA = {
+    "type": "string",
+    "enum": ["performance", "mixed"]
+}
+
+QUANTIZER_GROUP_PROPERTIES = {
+    **QUANTIZER_CONFIG_PROPERTIES,
+    **SCOPING_PROPERTIES,
+    "logarithm_scale": with_attributes(BOOLEAN,
+                                       description="Whether to use log of scale as the optimization parameter "
+                                                   "instead of scale itself. This serves as an optional regularization "
+                                                   "opportunity for training quantizer scales.",
+                                       default=QUANTIZATION_LOGARITHM_SCALE),
+}
+
+WEIGHTS_GROUP_SCHEMA = {
+    "type": "object",
+    "properties": {
+        **QUANTIZER_GROUP_PROPERTIES,
+    },
+    "additionalProperties": False
+}
+ACTIVATIONS_GROUP_SCHEMA = {
+    "type": "object",
+    "properties": {
+        **QUANTIZER_GROUP_PROPERTIES,
+        "unified_scale_ops": with_attributes(UNIFIED_SCALE_OP_SCOPES_SPECIFIER_SCHEMA,
+                                             description="Specifies operations in the model which will share "
+                                                         "the same quantizer module for activations. This "
+                                                         "is helpful in case one and the same quantizer scale "
+                                                         "is required for inputs to the same operation. Each "
```

Comment:
```suggestion
                                                         "is helpful in case the same quantizer scale "
                                                         "is required for each input to some operation. Each "
```
I get where "one and the same" comes from, but sadly it's not translated directly :) I rephrased it, hopefully the meaning is the same.